main.py

```diff
@@ -1,10 +1,42 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as json_file:
+        players_data = json.load(json_file)
+
+    for player in players_data:
+        race = players_data[player].get("race")
+        if race and not Race.objects.filter(name=race["name"]).exists():
+            Race.objects.create(name=race["name"],
+                                description=race["description"])
+        skills = race["skills"]
+        if skills:
```

Comment:
could u use get method here, with blank list as default value ?  to remove 18 line